src/platform/nxp/crypto/se05x/CHIPCryptoPALHsm_se05x_utils.h

```diff
@@ -74,13 +74,25 @@ extern "C" {
 /* Open session to se05x */
 CHIP_ERROR se05x_sessionOpen(void);
 
+/* Check if key exists in se05x */
+CHIP_ERROR Se05xCheckObjectExists(uint32_t keyid);
+
 /* Delete key in se05x */
 void se05x_delete_key(uint32_t keyid);
 
 /* Set key in se05x */
 CHIP_ERROR se05x_set_key_for_spake(uint32_t keyid, const uint8_t * key, size_t keylen, sss_key_part_t keyPart,
                                    sss_cipher_type_t cipherType);
 
+/* Set certificate in se05x */
```

Comment:
These comments seem redundant ... is there anything we can say that helps reading the code instead of re-reading it? I.e. `se05xSetCertificate` is obviously `se05x Set Certificate` i.e. `Sets the certificate in the se05x subsystem`.

Maybe you can talk about lifetime of buffer or what keyId is (not "keyId - the id of the key" but rather how does one get a get the key/create it and so on).

Other methods here are `se05x_snake_case` but these are `se05xUpperCamelCase` .. why the difference? Should it be consistent? Our code rule is generally "when in rome do what the romans do"...